mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowProtobufMapperTest.java

```diff
@@ -0,0 +1,39 @@
+package org.mlflow.tracking;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+
+import com.beust.jcommander.internal.Maps;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import org.mlflow.api.proto.Service;
+
+public class MlflowProtobufMapperTest {
+  @Test
+  public void testSerializeCamelCase() {
```

Comment:
Where is the camelCase within this test? Do we expect the intermediate `result` string to contain camelCase keys and or values?